addons/storyshots/storyshots-core/README.md

```diff
@@ -307,9 +307,32 @@ initStoryshots({
 
 Take a snapshot of a shallow-rendered version of the component. Note that this option will be overriden if you pass a `renderer` option.
 
-### `getSnapshotFileName`
```

Comment:
It's probably worth mentioning in `MIGRATION.md`, as well as puppeteer moving to other package